content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/tutorial-k8s.md

```diff
@@ -1,227 +1,282 @@
 ---
-date: '2021-09-21'
-title: "Deploying to a Local Kubernetes Cluster"
+date: '2021-11-14'
+title: "Deploying a Kubernetes Cluster"
 menu:
   corda-5-alpha:
     parent: corda-5-alpha-tutorials-deploy
     identifier: corda-5-alpha-tutorial-deploy-k8s
     weight: 1000
 section_menu: corda-5-alpha
 ---
-This section describes how to build and deploy Corda to a Kubernetes cluster on your local development machine.
 
-## Create a Kubernetes Cluster
+This page describes how to deploy Corda 5 Alpha. It assumes all necessary [prerequisites](../prerequisites.html) have been installed.
 
-The following instructions assume that you have a single-node Kubernetes cluster running with a Docker daemon.
-Two options that meet these requirements and have been tested with these instructions are:
+## Download and Register Docker Images
```

Comment:
I know it's a registry but I've never heard anyone use the verb `register` - `push` is the action in the Docker CLI.
```suggestion
## Download and push Docker images to registry
```